code/src/containers/tables/users-table/components/users-table-body/UsersTableBody.tsx

```diff
@@ -0,0 +1,60 @@
+import { UsersTableBodyProps } from "@/containers/tables/users-table/UsersTable.types";
+import {
+  roleBadges,
+  statusBadges
+} from "@/containers/tables/users-table/components/users-table-body/UsersTableBody.constants";
+
+import AppBadge from "@/components/app-badge/AppBadge";
+import { AppTableCell } from "@/components/app-table/components";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import formatDate from "@/utils/format-date/formatDate";
+
+const UsersTableBody = ({ user }: UsersTableBodyProps) => {
+  const { firstName, lastName, role, status, createdAt, email } = user;
+  const fullName = `${firstName} ${lastName}`;
```

Comment:
A full name is also used in another place. Maybe create function for contact first and last names? You can create the separate issue for that

Reply:
I created util and reused it